core/chains/evm/config/v2/defaults/Optimism_Alpha.toml

```diff
@@ -0,0 +1,16 @@
+ChainID = '28528'
+ChainType = 'optimism'
+FinalityDepth = 200
+LinkContractAddress = '0xdc2CC710e42857672E7907CF474a69B63B93089f'
+
+NoNewHeadsThreshold = '60s'
+TxResendAfterThreshold = '30s'
+
+[GasEstimator]
+EIP1559DynamicFees = true
+
+[GasEstimator.BlockHistory]
+BlockHistorySize = 24
```

Comment:
Why this value?

Reply:
It was an attempt to bypass the issue with the estimator and the unrecognized tx type we discussed. We can change the value after our tests.